workflows/genome_annotation/lncRNAs_annotation/Galaxy-Workflow-lncRNAs_annotation_workflow-tests.yml

```diff
@@ -0,0 +1,58 @@
+- doc: Test outline for Galaxy-Workflow-lncRNAs_annotation_workflow.ga
+  job:
+    Genome assembly:
+      class: File
+      location: https://zenodo.org/records/11367439/files/genome_assembly.fasta
+      filetype: fasta
+    Genome annotation:
+      class: File
+      location: https://zenodo.org/records/11367439/files/genome_annotation.gff3
+      filetype: gff3
+    RNA-Seq:
```

Comment:
please call this something like "Mapped reads from RNA-seq" or "Alignments from RNA-seq".